tests/shared/src/test/scala/epollcat/TcpSuite.scala

```diff
@@ -97,12 +97,12 @@ class TcpSuite extends EpollcatSuite {
           _ <- IO(assertEquals(res, "pong"))
         } yield ()
 
-        serverCh.bind(new InetSocketAddress(0)) *> server.both(client).void
+        serverCh.bind(new InetSocketAddress("localhost", 0)) *> server.both(client).void
     }
   }
```

Comment:
>  I find it useful to have the IPv6 tests in their own Suite.

Yes, this is a very good idea. In fact, we should also considering creating completely isolated sbt modules: one for testing `preferIPv4Stack = true` and another for testing `preferIPv4Stack = false`. Since this can be initialized only once, even separate suites is not sufficient insulation.